doc/rbd/rados-rbd-cmds.rst

```diff
@@ -118,24 +118,32 @@ For example::
 Retrieving Image Information
 ============================
 
-To retrieve information from a particular image, execute the following,
-but replace ``{image-name}`` with the name for the image:: 
+To retrieve information from a particular image, run the following command, but
+replace ``{image-name}`` with the name for the image:
 
-	rbd info {image-name}
+.. prompt:: bash $
+
+   rbd info {image-name}
 	
-For example::
+For example:
+
+.. prompt:: bash $
 
-	rbd info foo
+   rbd info foo
 	
-To retrieve information from an image within a pool, execute the following,
-but replace ``{image-name}`` with the name of the image and replace ``{pool-name}``
-with the name of the pool:: 
+To retrieve information from an image within a pool, run the following command,
+but replace ``{image-name}`` with the name of the image and replace
+``{pool-name}`` with the name of the pool:
 
-	rbd info {pool-name}/{image-name}
+.. prompt:: bash $
 
-For example:: 
+   rbd info {pool-name}/{image-name}
```

Comment:
At a former employer we had the convention of naming RBD images with `userid/<uuid>`, which collided with this syntax in a rather annoying way.  I wonder if we might do well to advise instead (or at least additionally) e.g. `rbd -p {pool-name| info {image-name}`

Reply:
It's a good point. I'll incorporate it.